ninja/orm/factory.py

```diff
@@ -55,7 +55,7 @@ def create_schema(
         if key in self.schemas:
             return self.schemas[key]
 
-        model_fields_list = self._selected_model_fields(model, fields, exclude)
+        model_fields_list = list(self._selected_model_fields(model, fields, exclude))
```

Comment:
@vitalik I added a cast to `list` here to fix this bug:
If `optional_fields` was equal to `__all__`, `optional_fields = [f.name for f in model_fields_list]` a few lines below was exhausting the `model_fields_list` iterator, and made `for fld in model_fields_list` iterate over an empty list, producing an empty Pydantic model.